docs/changelog.md

```diff
@@ -3,16 +3,79 @@ v0.27 + 1
 
 ### Changes or improvements
 
+* HTTP proxy support is now builtin; libcurl is no longer used to support
+  proxies and is removed as a dependency.
+
+* Certificate and credential callbacks can now return `GIT_PASSTHROUGH`
+  to decline to act; libgit2 will behave as if there was no callback set
+  in the first place.
+
 * The line-ending filtering logic - when checking out files - has been
   updated to match newer git (>= git 2.9) for proper interoperability.
 
+* Symbolic links are now supported on Windows when `core.symlinks` is set
+  to `true`.
+
 * Submodules with names which attempt to perform path traversal now have their
   configuration ignored. Such names were blindly appended to the
   `$GIT_DIR/modules` and a malicious name could lead to an attacker writing to
   an arbitrary location. This matches git's handling of CVE-2018-11235.
 
+* Object validation is now performed during tree creation in the
+  `git_index_write_tree_to` API.
+
+* Configuration variable may now be specified on the same line as a section
+  header; previously this was erroneously a parser error.
+
+* When an HTTP server supports both NTLM and Negotiate authentication
+  mechanisms, we would previously fail to authenticate with any mechanism.
+
+* The `GIT_OPT_SET_PACK_MAX_OBJECTS` option can now set the maximum
+  number of objects allowed in a packfile being downloaded; this can help
+  limit the maximum memory used when fetching from an untrusted remote.
+
+* Line numbers in diffs loaded from patch files were not being populated;
+  they are now included in the results..
+
+* The repository's index is reloaded from disk at the beginning of
+  `git_merge` operations to ensure that it is up-to-date.
+
+* Mailmap handling APIs have been introduced, and the new commit APIs
+  `git_commit_committer_with_mailmap` and `git_commit_author_with_mailmap`
+  will use the mailmap to resolve the committer and author information.
+  In addition, blame will use the mailmap given when the
+  `GIT_BLAME_USE_MAILMAP` option.
+
+* Ignore handling for files in ignored folders would be ignored.
+
+* Worktrees can now be backed by bare repositories.
+
+* Trailing spaces are supported in `.gitignore` files, these spaces were
+  previously (and erroneously) treated as part of the pattern.
+
+* The library can now be built with mbedTLS support for HTTPS.
```

Comment:
I think that was in 0.26 ?

Reply:
I don't think so?  We didn't actually end up merging it until early 2018 and I don't see mbedTLS in the `maint/v0.27` branch...?